workspaces/services/guacamole-azure-dsvm/porter.yaml

```diff
@@ -0,0 +1,87 @@
+name: tre-service-guacamole-win-dsvm
+version: 0.1.2
+description: "An Azure TRE User Resource Template for Guacamole (Windows DSVM)"
+registry: azuretre
+dockerfile: Dockerfile.tmpl
+
+credentials:
+  - name: azure_tenant_id
+    env: ARM_TENANT_ID
+  - name: azure_subscription_id
+    env: ARM_SUBSCRIPTION_ID
+  - name: azure_client_id
+    env: ARM_CLIENT_ID
+  - name: azure_client_secret
+    env: ARM_CLIENT_SECRET
+
+parameters:
+  - name: workspace_id
+    type: string
+  - name: tre_id
+    type: string
+  - name: mgmt_resource_group_name
+    type: string
+    description: "Resource group containing the shared ACR"
+    env: MGMT_RESOURCE_GROUP_NAME
+  - name: parent_service_id
+    type: string
+    description: "Resource group containing the shared ACR"
+    env: PARENT_SERVICE_ID
+  - name: tfstate_resource_group_name
+    type: string
+    description: "Resource group containing the Terraform state storage account"
+  - name: tfstate_storage_account_name
+    type: string
+    description: "The name of the Terraform state storage account"
+  - name: tfstate_container_name
+    env: tfstate_container_name
+    type: string
+    default: "tfstate"
+    description: "The name of the Terraform state storage container"
+  - name: arm_use_msi
+    env: ARM_USE_MSI
+    default: true
```

Comment:
Is this defaulting to true on the other bundles? I think it is false, unless it is explicitly set to false in the make file when running locally.  Don't mind either way as long as all consistent.

Reply:
others are false. I thought true is needed as it usually will work with MSI. 
I will switch